src/plugins/vuetify.js

```diff
@@ -33,19 +33,234 @@ export default function vuetifyConstructor (i18n){
                     darkGreen: '#4F665A',
                     streamerMode: '#9146ff',
                     error: '#ff5252',
+                    background: '#ded3af',
+                    home: '#ded3af',
+                    header: '#f1e9d6',
+                    card: '#f1e9d6',
+                    notepad: '#fafafa',
+                    page: '#ded3af',
+                    content: '#f1e9d6',
+                    footer: '#7fad94',
+                    scrollbar: '#3e3e3e',
+                    gmap: []
+                },
+                dark: {
+                    primary: '#468F69',
+                    secondary: '#808F87',
+                    accent: '#D0CABC',
+                    beige: '#ded3af',
```

Comment:
Small issue the country mode ;)
![image](https://user-images.githubusercontent.com/20130405/204102702-6e4a8dad-f972-469f-8d89-000fe2b5d0c0.png)
```suggestion
                    beige: '#181818',
```